features/application-mgt/org.wso2.carbon.identity.application.mgt.server.feature/resources/identity/authenticationTemplates/pushDeviceEnrollment.json

```diff
@@ -0,0 +1,58 @@
+{
+  "category": "ProgressiveEnrollment",
+  "name": "Push Device Progressive Enrollment",
+  "title": "Push Notification Device Progressive Enrollment Template",
+  "summary": "If Push Authenticator is set as a 1st authentication factor and there is a need to enable progressive Passkey enrollment, this adaptive script, along with the relevant connections configurations, should be added.",
```

Comment:
Here, we have referred to progressive enrollment as `progressive Passkey enrollment`. Was this intentional for push?

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6461/commits/ee943e1732f3061cfb26f7102fa43e13701c2ac3